src/components/navbars/DesktopNav.jsx

```diff
@@ -1,8 +1,9 @@
+import { useNavigate } from 'react-router-dom';
```

Comment:
Is the empty div `<div></div>` on line 51 necessary?

Reply:
Yes, it is needed, without it the buttons moves when you go to the search page.